config/monitoring/metrics/opencensus/100-ocagent.yaml

```diff
@@ -0,0 +1,104 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Based on https://github.com/census-instrumentation/opencensus-service/raw/master/example/k8s.yaml
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: opencensus-agent-config
+  namespace: knative-monitoring
+  labels:
+    serving.knative.dev/release: devel
+data:
+  opencensus-agent-config: |
+    receivers:
+      opencensus:
+        address: ":55678"
+    #   jaeger: {}
+    #   zipkin: {}
+    exporters: {}
+    # stackdriver:
+    #   project: "my-gcp-project"  # Optional, defaults to node project
+    #   metric_prefix: "your-metric-prefix"
+    #   enable_metrics: true
+    #   enable_tracing: true
+    # prometheus:
+    #   namespace: "namespace"
```

Comment:
FYI, the effect of the "namespace" field is it prefixes all the metrics names with "namespace_" (i.e. request_count -> namespace_request_count). We normally use this field to pass the component name as an option to the embedded Prometheus exporter such as "autoscaler" or "activator".

It looks like we'll lose the component names in the Prometheus metrics with the opencensus agent exporter. Though we can get them back by specifying the view.Name field on each view at register time like:

```
		&view.View{
			Name:        path.Join(component, desiredPodCountM.Name()), // defaults to Measure.Name if not specified
			Description: "Number of pods autoscaler wants to allocate",
			Measure:     desiredPodCountM,
			Aggregation: view.LastValue(),
			TagKeys:     metrics.CommonRevisionKeys,
		},
```